packages/core/admin/admin/src/content-manager/components/RelationInput/tests/RelationInput.test.js

```diff
@@ -95,7 +95,6 @@ describe('Content-Manager || RelationInput', () => {
     expect(screen.getByText('Some Relation')).toBeInTheDocument();
     expect(screen.getByText('Load more')).toBeInTheDocument();
     expect(screen.getByText('Select...')).toBeInTheDocument();
-    expect(screen.getByText('Relation 1')).toBeInTheDocument();
```

Comment:
I'd like to keep this one but with the tooltip Relation 1 text appears twice, I'm not sure how to handle it in a clean way, any idea @gu-stav ? 👀

Reply:
I think you have to use `getAllByText` then, or reduce the scope of the selector, e.g. to exclude the tooltip element (by using `queryAllBy` with a selector e.g.)

Reply:
I tried `getAllByText` but wasn't sure what to do with that array to get the one I wanted 🤔
`screen.logTestingPlaygroundURL` helped, getting it by role link 👌